.config/forest.dic

```diff
@@ -33,3 +33,9 @@ tuple
 validator/S
 mempool
 sortition
+struct/S
+serializer/SM
+deserialize/DG
+deserializer/SM
+boolean
+baz
```

Comment:
What does `baz` mean?

Reply:
Can we simplify the wording of the documentation to avoid adding this word?

Reply:
Sure!

Reply:
Reworded the conflicting documentation from "foo/bar/baz" to "path/to/file"